iceoryx_examples/waitset/ice_waitset_trigger.cpp

```diff
@@ -53,7 +53,7 @@ class MyTriggerClass
     //            or copied. To support that we have to inform the waitset about
     //            our new origin, otherwise the WaitSet would end up in the wrong
     //            memory location when it calls the `hasTriggerCallback` with the
-    //            old origin (already moved) origin pointer. The same goes for
+    //            old origin (already moved) pointer. The same goes for
```

Comment:
```suggestion
    //            old origin (already moved) pointer. The same applies to
```

Reply:
Fixed because I had to rerun the pipeline anyway